### ISSUE TITLE
Some errors in data.csv

### ISSUE THREAD

**Issue by andersx** [ghutchis/conformer-scoring#2]
Thanks for uploading this! Just one thing I noticed when I tried to parse the data: there are a couple of entries in `data.csv` which look like:

```
Neutral_jobs/astex_1opk, rmsd205-opt, error <type 'str'> <type 'float'> <type 'float'>
```
Should these just be excluded from the dataset?

Thanks,
Anders

**Comment by ghutchis** [comment]
Huh, I thought we fixed that one. Let me check - this may be an older version of the database.

**Comment by ghutchis** [comment]
Those were problematic DFT optimizations. Should be fixed by commit 4b1c9280f

I also added some cases where there were MMFF optimized geometries with DFT. Some `nan` in the DFT column, but generally more data.

**Comment by andersx** [comment]
Thanks a lot! Looks like you can close this one.
